homeassistant/components/currencylayer/sensor.py

```diff
@@ -73,7 +73,7 @@ def unit_of_measurement(self):
     @property
     def name(self):
         """Return the name of the sensor."""
-        return self._base
+        return self._quote
```

Comment:
This change doesn't make sense as the target currency is used as unit of measurement. Now it would no longer be possible to see which the source currency is.

Reply:
The current situation is that it uses the source currency for all quotes. As as alternative, can we concatenate the source and target currencies?

Configuration:
```yaml
  - platform: currencylayer
    api_key: !secret currency_layer_api_key
    base: USD
    quote:
      - INR
      - ZAR
```

States:
![image](https://user-images.githubusercontent.com/20719501/71972276-5a1fab80-3232-11ea-933d-4c6006ccf7bf.png)